### ISSUE TITLE
Runtime Segmentation fault

### ISSUE THREAD

**Issue by ProkopHapala** [Luthaf/MOPAC7#1]
I compiled it according to readme, than I downloaded the file `formic_acid.mop` from here http://openmopac.net/manual/examples.html and by running

`/home/prokop/git/MOPAC7/Build/mopac formic_acid.mop` 

and I got an error. The same error I got for .mop files generated by Avogadro:

```
Program received signal SIGSEGV: Segmentation fault - invalid memory reference.

Backtrace for this error:
#0  0x7F7BF8378E08
#1  0x7F7BF8377F90
#2  0x7F7BF7AAA49F
#3  0x7F7BF7BE7285
#4  0x401C6D in MAIN__ at mopac.F90:?
Segmentation fault (core dumped)
```

**Comment by Luthaf** [comment]
Hello, and thank you for reporting this issue.

Unfortunately, I don't use MOPAC at all, I just converted the old build system to allow people to play with this old code. I never tried to run the code, so there might be bugs inside (the same ones as in the original code!).

Pinging @andersx and @jhjensen2 here, as I think they had the same issue of code compiling but not running.

**Comment by jhjensen2** [comment]
Yes, we had the same issue.  @charnley got MOPAC 7 downloaded here (http://bioinformatics.org/ghemical/download/current/) to run

**Comment by ProkopHapala** [comment]
in any case, I just tried that this one works well for me
https://sourceforge.net/projects/mopac7/

also it is converted into C, which I like, nevertheless the code is not readable at all (no sure if it is because of f2c or it is inherent non-readability of original fortran77)
